README.md

```diff
@@ -58,16 +76,15 @@ See [here](https://docs.ansible.com/ansible/devel/dev_guide/developing_collectio
 
 ### Communication
 
-We have a dedicated Working Group for Ansible development.
+We announce important development changes and releases through the [Ansible Bullhorn newsletter](http://eepurl.com/gZmiEP). If you are a collection developer, be sure you are subscribed.
+
+Join us in the `#ansible` (general use questions and support), `#ansible-community` (community and collection development questions), and other [IRC channels](https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels).
 
-You can find other people interested on the following [Libera.chat](https://libera.chat/) IRC channels -
-- `#ansible` - For general use questions and support.
-- `#ansible-devel` - For discussions on developer topics and code related to features or bugs in ansible-core.
-- `#ansible-community` - For discussions on community topics and community meetings, and for general development questions for community collections.
+We take part in the global quarterly [Ansible Contributor Summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track the [Bullhorn newsletter](https://github.com/ansible/community/issues/546) and join us.
```

Comment:
```suggestion
We take part in the global quarterly [Ansible Contributor Summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track the [Ansible Bullhorn newsletter](https://github.com/ansible/community/issues/546) and join us.
```

Reply:
or `Track Ansible's The Bullhorn newsletter ...`? Might be clearer, since the newsletter isn't called "Ansible Bullhorn", but "The Bullhorn".

Reply:
I changed the first mentioning of The Bullhorn to `Ansibles [The Bullhorn newsletter]..` and left this sentence almost as it was. Not to repeat `Ansible's`. Thanks!

Reply:
Aha, that was the reason I made this suggestion :)
I thought they should be identical. Feel free to ignore or change it to something else.

Reply:
thanks @felixfontein for clarifying:)